README.md

```diff
@@ -1980,3 +1981,9 @@ He's so good that I uploaded a couple of photos
 ---
 =======
 
+##Hi, my name [Anton](http://github.com/AntonKottans)
+I hope learning frontend with your community will feel easier, cos redux is really hard to understand at first sight. I don't like pictures or cats, so this is how april looks like in Saint Petersburg:
+![Spring](/gifs/usual_spring_in_saint_petersburg_2400.jpg)
```

Comment:
I guess you made a mistake in the path of your file.  That's why your picture is not displayed https://github.com/AntonKottans/mock-repo/tree/el_branch
The folder "gifs" - does not exist on this repo. Maybe will be correctly  - to change this naming of the folder to "gif"? See for example here https://github.com/kottans/mock-repo/pull/562/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R1887
Could you try the once again please?